service-deployment/src/grafana/start_server.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+chown -R grafana:grafana /var/lib/grafana /var/log/grafana
+
+if [ ! -z ${GF_AWS_PROFILES+x} ]; then
```

Comment:
>GF_AWS_PROFILES [](start = 12, length = 15)

what to get this value?

Reply:
It's the parameter set when running the image https://github.com/grafana/grafana/blob/80903c162e88e21c1fd9645ac38cf4bdee916287/docs/sources/installation/docker.md 
I will test whether the docker image works well without these codes.

Reply:
no need to do that. is it also a env. passed by docker? 

---
In reply to: [157507442](https://github.com/Microsoft/pai/pull/102#discussion_r157507442) [](ancestors = 157507442)